oracle-plugin/src/e2e-test/resources/pluginParameters.properties

```diff
@@ -55,3 +55,54 @@ longVarcharColumns=(ID VARCHAR2(100) PRIMARY KEY, COL1 LONG VARCHAR)
 longVarcharColumnsList=(ID,COL1)
 longVarcharValues=VALUES ('User1','48692054686572652120486F772061726520796F75206665656C696E67AbCdEF646179203F')
 outputDatatypesSchema4=[{"key":"ID","value":"string"},{"key":"COL1","value":"string"}]
+
+#ORACLE Invalid Properties
+invalidRef=invalidRef&^*&&*
+oracleInvalidTable=invalidTable
+invalidDatabaseName=invalidDB%$^%*
+invalidImportQuery=select
+invalidTableNameImportQuery=select * from abc;
+invalidDriverName=abcdriver
+invalidHost=testhost
+invalidPort=testPort
+invalidUserName=testUser
+invalidPassword=testPassword
+invalidBoundingQuery=SELECT MIN(id),MAX(id) FROM table
+invalidTable=table
+
+#ORACLE Valid Properties
+connectionArgumentsList=[{"key":"queryTimeout","value":"-1"}]
+batchValue=40
+rowPrefetch=60
+numberOfSplits=2
+zeroValue=0
+splitByColumn=ID
+oracleImportQuery=select * from HR.SOURCETABLE_ACLCHFZYSA where $CONDITIONS
+
+#bq properties
+projectId=cdf-athena
+dataset=test_automation
+bqOutputDatatypesSchema=[{"key":"ID","value":"decimal"},{"key":"LASTNAME","value":"string"}]
+jdbcUrl = jdbc:bigquery://https://www.googleapis.com/bigquery/v2:443;ProjectId=%s;OAuthType=3;
+
+#bq macro properties
+bqTruncateTable=true
+bqUpdateTableSchema=true
+
+#bq queries file path
+CreateBQTableQueryFile=testdata/BigQuery/BigQueryCreateTableQuery.txt
+InsertBQDataQueryFile=testdata/BigQuery/BigQueryInsertDataQuery.txt
+
+#ORACLE Datatypes
+datatypeColumns1=(COL23 FLOAT(4), COL28 TIMESTAMP, COL29 TIMESTAMP(9), COL30 TIMESTAMP WITH TIME ZONE, \
+  COL31 INTERVAL DAY(6) TO SECOND (5), COL32 INTERVAL YEAR(4) TO MONTH, COL33 DATE, COL34 BINARY_FLOAT, \
+  COL35 BINARY_DOUBLE)
+datatypeColumnsList1=(COL23,COL28,COL29,COL30,COL31,COL32,COL33,COL34,COL35)
+datatypeValues1=VALUES (1234.5679,TIMESTAMP'2023-01-01 2:00:00',TIMESTAMP'2023-01-01 2:00:00',\
+  TIMESTAMP'2023-01-01 2:00:00 -08:00',TIMESTAMP '2001-09-03 12:47:00.000000'- TIMESTAMP '2001-09-03 13:13:00.000000',\
+  INTERVAL '5-2' YEAR TO MONTH,TIMESTAMP '2023-01-01 00:00:00.000000',339999992740149460000,\
+  34000000000000000000000000000000000000000)
+outputDatatypesSchema1=[{"key":"COL23","value":"double"},{"key":"COL28","value":"timestamp"},\
```

Comment:
There are some changes done here in the recent code.
ORACLE TIMESTAMP : CDAP Datetime
ORACLE TIMESTAMP WITH TIME ZONE : CDAP Timestamp
ORACLE TIMESTAMP WITH LOCAL TIME ZONE : CDAP Datetime

Please change this accordingly.

Reply:
Change done.